api/src/main/java/io/strimzi/api/kafka/model/KafkaBridgeSpec.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.fabric8.kubernetes.api.model.ResourceRequirements;
+import io.strimzi.api.kafka.model.template.KafkaBridgeTemplate;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.Minimum;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "replicas", "bootstrapServers", "authentication",
+        "tls", "http", "image", "resources", "jvmOptions",
+        "logging", "metrics", "template"})
+@EqualsAndHashCode
+public class KafkaBridgeSpec implements UnknownPropertyPreserving, Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private int replicas;
+    private String bootstrapServers;
+
+    private KafkaBridgeAuthentication authentication;
+    private KafkaBridgeTls tls;
+    private Http http;
+    private String image;
+    private ResourceRequirements resources;
+    private JvmOptions jvmOptions;
+    private Logging logging;
+    private Map<String, Object> metrics;
+    private Map<String, Object> additionalProperties = new HashMap<>(0);
+    private KafkaBridgeTemplate template;
```

Comment:
Slightly related to what @scholzj said above and related to this issue https://github.com/strimzi/strimzi-kafka-bridge/issues/184 we should have even liveness and readiness probes configuration if we will implement the 184 issue. Let's remember about that.